### PR TITLE
added instructions for cloning testharness.js as a submodule

### PR DESCRIPTION
**Pull request by rhauck**
@tobie or @jgraham - can you review this? Thanks!

### CODE REVIEW

**Comment by rhauck** [comment]
Ok, incorporated your feedback @tobie.  I'll follow up on the list with @plinss to confirm the subrepo can be added to the css repo.  This should be ready to merge.

File: _layouts/default.html [hunk 1]
```diff
@@ -102,7 +102,7 @@
     }
     var $toc = $("#toc");
     if ($toc.length) {
-      var $headers = $("*[role=main] h2");
+      var $headers = $("*[role=main] h2, h3");
```
Comment (tobie) [inline _layouts/default.html]: Please chang `"*[role=main] h2, h3"` to `"*[role=main] h2, *[role=main] h3"`.

File: docs/testharness-tutorial.md [hunk 2]
```diff
@@ -22,14 +22,9 @@ created in JavaScript directly.
...
-  TODO This should describe how to include testharness.js as a submodule in 
-  web-platform-tests.
--->
-
 Let's get started with this code! The first thing you need to do to load
 `testharness.js` is to include it from a `script` element in the usual way.
-You can either download your own copy and set it up locally whichever way you
+You can either [download your own copy][1] and set it up locally whichever way you
 want, or if you're writing a test for a W3C service you should just point to
 the W3C copy:
```
Comment (tobie) [inline docs/testharness-tutorial.md:34]: That part is still suboptimal. Please either fix or don't delete the TODO.

**Comment by tobie** [comment]
Other than that, LGTM.

File: docs/testharness-tutorial.md (hunk 2 above)
Comment (rhauck) [inline docs/testharness-tutorial.md:34]: Yeah, that whole sentence is rather screwy, IMO.  I put the TODO back even though this PR satisfies what the original TODO said.  I'll try to revisit this later.

Comment (tobie) [inline docs/testharness-tutorial.md:34]: Yeah, my inline TODO's aren't the most explicit ever. :/

**Comment by rhauck** [comment]
Merging since 508aa7d covers the rest of @tobie 's feedback.

**Comment by tobie** [comment]
LGTM.

**Comment by tobie** [comment]
Did you merge this in the end? Confused.

**Comment by rhauck** [comment]
Well I thought I did yesterday - hence, my comment.  Now looking at, gh is telling me I closed it 39 minutes ago, which would have been when I deleted the branch.  Apparently I did not merge and I blew away the branch.  I'll send it in again shortly. Sorry about that.

**Pull request closed without merging**